src/server/scripts/Northrend/IcecrownCitadel/instance_icecrown_citadel.cpp

```diff
@@ -1002,9 +1002,15 @@ class instance_icecrown_citadel : public InstanceMapScript
                         // set the platform as active object to dramatically increase visibility range
                         // note: "active" gameobjects do not block grid unloading
                         if (GameObject* precipice = instance->GetGameObject(ArthasPrecipiceGUID))
+                        {
                             precipice->setActive(state == IN_PROGRESS);
```

Comment:
active should be removed here - being the author of this script i guarantee it

Reply:
Only for precipice, or for the platform too?

Reply:
both

Reply:
Done, thanks!